tests/signing/sigstore_test.py

```diff
@@ -229,9 +224,8 @@ def test_sign_verify_mocked_prod(
         manifest = serializer.serialize(sample_model_folder)
         signature_path = tmp_path / "model.sig"
         self._sign_manifest(
-            manifest,
+            signing.Payload(manifest),
```

Comment:
rename `_sign_manifest` to `_sign_payload` if the first arg we're passing in is a payload?

Reply:
or would it be simpler to keep it `_sign_manifest` and still pass in the manifest.

and the first line of `_sign_manifest` could just be `payload = signing.Payload(manifest)`

Reply:
Changed to the second one